pkg/plugins/user_test.go

```diff
@@ -181,7 +224,7 @@ rancher:$6$2SMtYvSg$wL/zzuT4m3uYkHWO1Rl4x5U6BeGu9IfzIafueinxnNgLFHI34En35gu9evtl
 			defer cleanup()
 
 			err = User(l, schema.Stage{
-				Users: map[string]schema.User{"foo": {PasswordHash: `$fkekofe`, Homedir: "/home/foo", SSHAuthorizedKeys: []string{"github:mudler", "efafeeafea,t,t,pgl3,pbar"}}},
+				Users: map[string]schema.User{"foo": {PasswordHash: `$fkekofe`, Homedir: "/HomeDir()/foo", SSHAuthorizedKeys: []string{"github:mudler", "efafeeafea,t,t,pgl3,pbar"}}},
```

Comment:
I'll call this hardcore testing :D 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExeTNtZGJxM3NtNGo1bHJibjhyMXA1NzV2cGtvdDc2NGJ4eWp5dmdkZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Mxbs3AP2Y1wKA/giphy.gif)

Reply:
oh shoot, there's something off there actually, it's not that good hahaha, fixing but it's actually a good idea